### ISSUE TITLE
Can't install @vite-pwa/sveltekit

### ISSUE THREAD

**Issue by dongumayagay** [vite-pwa/sveltekit#13]
can't install @vite-pwa/sveltekit

Error message:
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@vite-pwa%2Fsveltekit: Not Found - 404
```
This error happened while installing a direct dependency of C:\Users\Maldita\Documents\Homework\my-app

@vite-pwa/sveltekit is not in the npm registry, or you have no permission to fetch it.

No authorization header was set for the request.
Progress: resolved 2, reused 2, downloaded 0, added 0
```
![image](https://user-images.githubusercontent.com/51516536/198150224-230e5e52-e54a-4fb9-a727-2808f3dad35d.png)

how to reproduce

1. create new sveltekit app according to https://kit.svelte.dev/
2. install @vite-pwa/sveltekit according to https://vite-pwa-org.netlify.app/frameworks/sveltekit.html

**Comment by Cluster2a** [comment]
The package has not been released yet.

**Comment by userquin** [comment]
released v0.0.1
